instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/Instrumenter.java

```diff
@@ -19,45 +17,50 @@
 import java.util.stream.Collectors;
 import java.util.stream.StreamSupport;
 
-public abstract class Instrumenter<REQUEST, RESPONSE> {
+public /* sealed */ class Instrumenter<REQUEST, RESPONSE> {
```

Comment:
```suggestion
public class Instrumenter<REQUEST, RESPONSE> {
```

sealed keyword is more for when the constructor has to be made public I believe